indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -596,7 +596,8 @@
    :variable-name (-> m/string-field-mapping m/stored m/doc-values)
    :variable-name.lowercase (m/doc-values m/string-field-mapping)
    :measurement (-> m/string-field-mapping m/stored m/doc-values)
-   :measurement.lowercase (m/doc-values m/string-field-mapping)})
+   :measurement.lowercase (m/doc-values m/string-field-mapping)
+   :keyword m/text-field-mapping})
```

Comment:
We should never aggregate or sort on keyword so I am fine with not making it a doc-values field.